examples/tree_1d_dgsem/elixir_burgers_shock.jl

```diff
@@ -0,0 +1,93 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the (inviscid) Burgers' equation
+
+equations = InviscidBurgersEquation1D()
+
+basis = LobattoLegendreBasis(3)
+# Use shock capturing techniques to supress oscillations at discontinuities
+indicator_sc = IndicatorHennemannGassner(equations, basis,
+                                         alpha_max=1.0,
+                                         alpha_min=0.001,
+                                         alpha_smooth=true,
+                                         variable=Trixi.scalar)
+
+# Use same flux for DG & FV in this example
+numerical_flux = flux_lax_friedrichs
+
+volume_integral = VolumeIntegralShockCapturingHG(indicator_sc;
+                                                 volume_flux_dg=numerical_flux,
```

Comment:
That's a bad idea - DG volume fluxes need to be symmetric...

Reply:
Symmetric in the sense that for a two-point flux $F(u_i, u_{i\pm 1}) = F(u_{i\pm 1}, u_i)$? In that case, using the central flux $$F_{i+\frac12} = F_\text{Central}(u_i, u_{i+1})= \frac{f(u_i) + f(u_{i+1})}{2}$$
would be fine, right?
Although using the central scheme is seldom a good idea...

Reply:
Yes to both. We usually use some entropy-conservative numerical flux for the DG volume integral - you may have a look at the tutorials or other examples using `VolumeIntegralShockCapturingHG`.